controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -311,7 +309,7 @@ export class OrganizationRepository {
       )
       .execute();
 
-    return userRoles.map((role) => role.role);
+    return [...new Set(userRoles.map((role) => role.role))];
```

Comment:
How is it possible that we get duplicates?

Reply:
yep, its a mistake